provisioner/powershell/provisioner.go

```diff
@@ -358,13 +369,22 @@ func (p *Provisioner) createFlattenedEnvVars(elevated bool) (flattened string) {
 	// Always available Packer provided env vars
 	envVars["PACKER_BUILD_NAME"] = p.config.PackerBuildName
 	envVars["PACKER_BUILDER_TYPE"] = p.config.PackerBuilderType
+
 	httpAddr := common.GetHTTPAddr()
 	if httpAddr != "" {
 		envVars["PACKER_HTTP_ADDR"] = httpAddr
 	}
 
+	// interpolate environment variables
```

Comment:
imo, we should just add an `envVars[PACKER_WINDOWS_PASSWORD"] = ...` statement above to set the password, instead of letting the user choose which env var to put it in.